### PR TITLE
Spelling Tool and associated fixes

### PR DESCRIPTION
**Pull request by samgnaniah**
This includes the spelling file and associated fixes.

### CODE REVIEW

File: en/docs/administer/managing-users-and-roles/managing-user-roles.md [hunk 1]
```diff
@@ -109,7 +109,7 @@ Follow the instructions below to create the `creator`, `publisher`, and `subscri
     These roles do not have any permissions assigned to it, but it is used to manage the visibility of the corresponding service provider that is created in the format of `'<username>_<applicationName>_PRODUCTION'` within the Key Manager. The created service provider is only visible to users with the latter mentioned role that has been generated automatically. Only if a user with admin privileges assigns the latter mentioned role to a user, will that user be able to view the details of the service provider that is created per application.
 
 !!! warning
-    As a limitation, when you delete a user and create another with the same username, applications of the previous user will be visible on the Dev portal applications listing page.
+    As a limitation, when you delete a user and create another with the same username, applications of the previous user will be visible on the Developer portal applications listing page.
```
Comment (Mariangela) [inline en/docs/administer/managing-users-and-roles/managing-user-roles.md:112]: Developer portal > Developer Portal

File: en/docs/consume/invoke-apis/invoke-apis-using-tools/include-additional-headers-in-the-api-console.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 
 !!! Note
-        The Swagger API Console is a JavaScript client that runs in the Developer Portal and makes JavaScript calls from the Developer Portal to the API Gateway. In order to successfuly invoke an API by specifying an additional header via the API console, first you must specify the header that you want to add, under the CORS ( [Cross Origin Resource Sharing](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) ) configuration.
+        The Swagger API Console is a JavaScript client that runs in the Developer Portal and makes JavaScript calls from the Developer Portal to the API Gateway. In order to successfully invoke an API by specifying an additional header via the API console, first you must specify the header that you want to add, under the CORS ( [Cross Origin Resource Sharing](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) ) configuration.
```
Comment (Mariangela) [inline en/docs/consume/invoke-apis/invoke-apis-using-tools/include-additional-headers-in-the-api-console.md:7]:
```suggestion
        The Swagger API Console is a JavaScript client that runs in the Developer Portal and makes JavaScript calls from the Developer Portal to the API Gateway. In order to successfully invoke an API by specifying an additional header via the API console, first you must specify the header that you want to add, under the CORS ([Cross Origin Resource Sharing](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS)) configuration.
```

File: en/docs/consume/invoke-apis/invoke-apis-using-tools/include-additional-headers-in-the-api-console.md [hunk 3]
```diff
@@ -4,7 +4,7 @@
...
 
 
         Open the `<API-M_HOME>/repository/conf/deployment.toml` file, and specify the additional headers ( `userId` , in this case)under the `[apim.cors]` section: Alternatively, you could choose to add this additional header only to a specific API ( [Enabling Cors per API]({{base_path}}/design/advanced-topics/enabling-cors-for-apis/#enabling-cors-per-api)).
```
Comment (Mariangela) [inline en/docs/consume/invoke-apis/invoke-apis-using-tools/include-additional-headers-in-the-api-console.md:10]: sentence is not correct

File: en/docs/consume/manage-application/generate-keys/generate-api-keys.md [hunk 4]
```diff
@@ -57,9 +57,9 @@ Let's assume that you are working in a production environment.
 !!! note
     If you need an Access Token with scopes, make sure that you have created the scopes in advance on the Okta side.
 
-## Generate application keys using Keycloak Key Manager
+## Generate application keys using KeyCloak Key Manager
```
Comment (Mariangela) [inline en/docs/consume/manage-application/generate-keys/generate-api-keys.md:60]: KeyCloak > Keycloak

https://en.wikipedia.org/wiki/Keycloak

**Review by Mariangela — APPROVED** [review]

File: en/docs/administer/managing-users-and-roles/managing-user-roles.md (hunk 1 above)
Comment (samgnaniah) [reply en/docs/administer/managing-users-and-roles/managing-user-roles.md:112]: Portal is correctly spelt so the tool does not flag it. Will fix manually.

File: en/docs/consume/manage-application/generate-keys/generate-api-keys.md (hunk 4 above)
Comment (samgnaniah) [reply en/docs/consume/manage-application/generate-keys/generate-api-keys.md]: Fixed.

File: en/docs/administer/managing-users-and-roles/managing-user-roles.md (hunk 1 above)
Comment (samgnaniah) [reply en/docs/administer/managing-users-and-roles/managing-user-roles.md]: Fixed.

File: en/docs/consume/invoke-apis/invoke-apis-using-tools/include-additional-headers-in-the-api-console.md (hunk 3 above)
Comment (samgnaniah) [reply en/docs/consume/invoke-apis/invoke-apis-using-tools/include-additional-headers-in-the-api-console.md]: Fixed.

**Pull request merged**